src/resources/entity-extraction.js

```diff
@@ -20,7 +19,7 @@ export default class EntityExtraction extends ApiResource {
       // Instead of foo[0]=bar&foo[1]=baz
       // (dimensions for example)
       useQuerystring: true,
-      qs: {
+      qs: JSON.parse(JSON.stringify({
```

Comment:
Why the need to clone this object ?

Reply:
because undefined values will be transmitted to the web service in the form `param=` which prevent the ws to use the default value. (probably, we should check it web-service side ?)

Reply:
Ok, fine by me 😄 
(This will work for `undefined` values, but not `null` or `''`)

Reply:
I will delete this PR. I managed to change it on the webserver side